File: src/Desafio.cs
using System;

public class Desafio {
    public static void Main(){

        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;

        if (valorSalario >= 0 && valorSalario <= 1100){
            valorImposto = 0.05f * valorSalario;
        } else if (valorSalario >= 1100.01 && valorSalario <= 2500.0){
            valorImposto = 0.10f * valorSalario;
        } else {
            valorImposto = 0.15f * valorSalario;
        }        

        float saida = valorSalario - valorImposto + valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));
    }
}

File: src/Desafio.js
const { stdin } = require('process');
const readline = require('readline')

const rl = readline.createInterface(
  process.stdin, process.stdout
)

rl.setPrompt('Entre com o valor do salario: ');
rl.prompt();
rl.on('line', (valorSalario) => {
  rl.close();
});
rl.setPrompt('Entre com o valor do beneficio: ');
rl.prompt('line', (valorBeneficios) => {
  rl.close();
});

const valorImposto = calcularImposto(valorSalario);

const saida = valorSalario - valorImposto + valorBeneficios;
print(saida.toFixed(2));

function calcularImposto(salario){
  let aliquota;
  if (salario >= 0 && salario <= 1100){
    aliquota = 0.05;
  } else if (salario >= 1100.01 && salario <= 2500.0) {
    aliquota = 0.10;
  } else {
    aliquota = 0.15;
  }

  return aliquota * salario;
}